.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: go
 
 go:
-  - 1.3
-  - 1.4
   - 1.5
-  - tip
```

Comment:
When I ran tests in travis few days ago "tip" version was 1.4.3 and it doesn`t support vendor expirement. I`ll check if travis supports 1.6 now.